### PR TITLE
Add desktop version

### PR DESCRIPTION
**Pull request by frempongdev**
In this Pull request, I made the following changes :
- Added Style and Changes to complete Desktop version.
- - Added a new screen query for Desktop.
- - Updated CSS file to fit Desktop query.
- - Edited HTML file to adjust additional desktop files.
- - Added unique files for both Mobile and Desktop versions.
- - Adjusted HTML files to accommodate changes in both Mobile and Desktop versions.
- - Ensured strict linter guidelines.

### CODE REVIEW

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @frempongdev 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -476,3 +486,481 @@
...
+  }
+
+  .footer-container {
+    background-color: #6070ff;
+    background-image: url(assets/desktop/contact\ form\ background\ shapes.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+    border-top-left-radius: 100px;
+    width: 100%;
+    height: 100%;
```
[the remark this answers is not in the document]
Comment (frempongdev) [reply style.css:905]: Kindly give me the screen size that shows this unwrapped section of the page. 
It seems to be working fine on all my screens.

File: index.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
       <a href="#">My Logo</a>
 
       <nav class="main-nav">
-        <a href="#">☰</a>
+        <a class="nav-icon" href="#">☰</a>
         <ul class="nav-wrapper">
           <li><a href="#">Portfolio</a></li>
           <li><a href="#">About</a></li>
```
[the remark this answers is not in the document]
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html [hunk 3]
```diff
@@ -124,21 +123,25 @@ <h4 class="grey-text">2015</h4>
                 </ul>
               </div>
             </div>
+            <div class="see-work">
+              <button>See Project</button>
+              <!-- <a href="#">See Project</a> -->
```
[the remark this answers is not in the document]
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html [hunk 4]
```diff
@@ -112,212 +113,247 @@ <h4 class="grey-text">Back End Dev</h4>
...
             <div class="desc-wrapper">
               <h2>Multi-Post<br />Stories</h2>
               <div class="dec-details">
-                <h4>CANOPY</h4>
+                <h4 class="mobile-text">CANOPY</h4>
+                <h4 class="desktop-text">FACEBOOK</h4>
                 <img class="mobile-dot" src="assets/mobile/dot.png" alt="dot" />
-                <h4 class="grey-text">Back End Dev</h4>
+                <h4 class="grey-text mobile-text">Back End Dev</h4>
+                <h4 class="grey-text desktop-text">Full Stack Dev</h4>
```
Comment (Abenezer-Tilahun) [inline index.html:153]: - [ optional ] As suggested by the previous reviewer Kindly make this text in one line and not broken in 2 lines to improve readability and to follow the design in Figma 💯.



The Figma design Look likes this  : 

Capture


Your Design Look likes this : 

Capture

File: style.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (Abenezer-Tilahun) [reply style.css]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @frempongdev 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️Good Job  Implement the desktop version with the usage of media queries.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**